package.json

```diff
@@ -32,7 +32,7 @@
     "body-parser": "1.15.1",
     "bootstrap": "3.3.6",
     "express": "4.13.4",
-    "ng2-translate": "2.0.0",
+    "ng2-translate": "2.1.0",
```

Comment:
Thanks for catching that. I forgot to clear node_modules before running npm install. Had the angular rc in there. I will revert that.